### DISCUSSION TITLE
Convert JSON to TypeScript format

### DISCUSSION THREAD

**Discussion by llity** [evan-liu/karabiner.ts#136] [Q&A]
I have a lot of settings written into a JSON file, can it be automatically converted to TS format?

**Comment by evan-liu** [comment]
Sorry, it is currently not supported. There are 2 ways to use the JSON though: 

1. [importJson() function](https://karabiner.ts.evanliu.dev/imports#import-json)
2. The JSON code can be used directly in karabiner.ts

```typescript
writeToProfile(
  'Default',
  [
    ...[rulesInTs],
    ...[rulesInJson],
  ]
)
```

Then migrate them to ts in steps.
